--- a/lib/app/theme.dart
+++ b/lib/app/theme.dart
@@ -75,8 +75,8 @@
   static TextStyle bodySmall(Color color) => getBodyFont(TextStyle(
     fontSize: 12, fontWeight: FontWeight.w400,
     color: color,
   ));

   static TextStyle labelLarge(Color color) => getDisplayFont(TextStyle(
     fontSize: 14, fontWeight: FontWeight.w600, letterSpacing: 0.1,
     color: color,
   ));
   static TextStyle labelSmall(Color color) => getDisplayFont(TextStyle(
     fontSize: 11, fontWeight: FontWeight.w500, letterSpacing: 0.5,
     color: color,
   ));
 }
